checkpoint_harmony_endpoint/README.md

```diff
@@ -0,0 +1,77 @@
+# Agent Check: checkpoint_harmony_endpoint
+
+## Overview
+
+Checkpoint Harmony Endpoint is a next-generation endpoint security solution designed to prevent, detect, and respond to threats on user devices (desktops, laptops, servers, etc). This integration monitors [Checkpoint Harmony Endpoint][1].
+
+## Setup
+1. Collecting logs is disabled by default in the Datadog Agent. Enable it in the `datadog.yaml` file with:
+
+    ```yaml
+      logs_enabled: true
+    ```
+2. Add this configuration block to your `checkpoint_harmony_endpoint.d/conf.yaml` file to start collecting your checkpoint_harmony_endpoint logs:
+
+    ```yaml
+      logs:
+        - type: file
+          path:  /var/log/checkpoint.log
+          source: checkpoint_harmony_endpoint
+          service: <SERVICE_NAME>
+    ```
+
+    Change the `path` and `service` parameter values for your environment.
+
+3. [Restart the Agent][4].
+
+### Installation
+
+The checkpoint harmony endpoint check is included in the [Datadog Agent][2] package.
+
+### Prerequisites
+
+1. Administrative access to Checkpoint Harmony Endpoint - Gaia installed on your server.
+2. The Datadog Agent installed and running (on a server or container that can receive syslog messages).
+3. Network Access between the endpoint and the Datadog Agent (usually port 514, but may be a custom value). Enable log exporter in the Smart Console for log streaming.
+4. Syslog support enabled in the Datadog Agent (with a TCP or UDP  listener configured).
+
+### Validation
+
+1. Confirm the Datadog Agent is listening on the correct port (`514` in the following examples)
+    `sudo netstat -tunlp | grep 514`
+    If using TCP and UDP listeners, use the following command:
+    `sudo lsof -i :514`
```

Comment:
~~~suggestion
1. Confirm the Datadog Agent is listening on the correct port (`514` in the following examples)
   ```sh
   sudo netstat -tunlp | grep 514
   ```
   
   If using TCP and UDP listeners, use the following command:
   ```sh
   sudo lsof -i :514
   ```
~~~